package.json

```diff
@@ -53,6 +53,13 @@
     "yazl": "^2.5.1"
   },
   "devDependencies": {
+    "@babel/cli": "^7.14.3",
```

Comment:
Should we use the same babel versions as our other tooling (see package-lock.json)?

Reply:
Everything looks nicely de-duped in the package-lock.json, and the build succeeds. So I'm not going to mess with it.